docs/custom_conf.py

```diff
@@ -57,6 +57,12 @@
 # features, it is sufficient to update the settings in the "Project
 # information" section.
 
+# FIXME: temporarily disable missing xrefs in this branch, until the missing
+# links to Juju and Charmcraft are added.
+suppress_warnings = [
+    'myst.xref_missing',
+]
+
```

Comment:
This should be the only bit that needs to be removed once the links to Juju/Charmcraft are working.